infra/infra.tf

```diff
@@ -89,9 +89,6 @@ resource "aws_amplify_branch" "main" {
   enable_notification = "false"
   framework           = "Vue"
   stage               = "DEVELOPMENT"
-  environment_variables = {
-    VITE_ORGANIZATIONS_CONTENT_SHEET_ID = "1Vp57HDXod3FgP2aZzDNvYYRBNgPbvNsqqxNJ2b-daoc"
```

Comment:
Do you still need the env variable for the production environment above? If not I'd say remove both for consistency.

Reply:
I missed that! Thanks!